plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -3333,6 +3935,94 @@ def reboot_factory_reset_function(self, ap_list, reboot_or_reset):
 
             return self
 
+    def bulk_ap_update(self, bulk_config):
+        """
+        Access Point bulk update common field to update to all mentioned AP list.
+
+        Parameters:
+            self (dict): A dictionary used to collect the execution results.
+            bulk_config (dict): A dict containing the APs mac address and cofig details which need to update APs.
+
+        Returns:
+            dict: A dictionary containing the result of the buld access point update status.
+        """
+        ap_exist, ap_details = self.get_accesspoint_details(bulk_config)
+        self.payload["access_point_details"] = ap_details
+        ap_update_list = []
+        common_config = {}
+        ap_output_list = []
+        if ap_exist and len(ap_details) > 0:
+            for each_ap in ap_details:
+                ap_config_exists, ap_configuration = self.get_accesspoint_config(each_ap["ap_ethernet_mac_address"])
+                self.log("Access point configuration exists: {0}, Current configuration: {1}"
+                         .format(ap_config_exists, self.pprint(ap_configuration)), "INFO")
+                self.want = bulk_config.get("common_fields_to_change")
+                self.want["mac_address"] = each_ap["mac_address"]
+                ap_name = [ap.get('ap_name') for ap in bulk_config.get("ap_identifier")
+                           if (each_ap["mac_address"] == ap.get('mac_address') or
+                               each_ap["hostname"] == ap.get('hostname') or
+                               each_ap["management_ip_address"] == ap.get('management_ip_address'))]
+                self.want["ap_name"] = ap_name[0]
```

Comment:
if ap_name is there, then we can add it.. Otherwise don't we put a log message?

            if ap_name:
                self.want["ap_name"] = ap_name[0]
                ap_output_list.append(ap_name[0])
                self.log("Identified AP name: {}".format(ap_name[0]), "INFO")
            else:
                self.log("No matching AP name found for MAC: {}".format(each_ap["mac_address"]), "WARNING")